### PR TITLE
Fixed ipv6 cidr on Regex

### PR DESCRIPTION
**Pull request by ansvu**

### CODE REVIEW

File: pkg/obfuscator/ip.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ const (
 var (
 	ipv4re = `(([1-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])[.]([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])[.]([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])[.]|([1-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])[_-]([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])[_-]([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])[_-])([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]){1,3}`
 	// ipv6re is not perfect. it can still catch words like :face:bad as a valid ipv6 address
-	ipv6re      = `([a-f0-9]{0,4}[:]){1,8}[a-f0-9]{1,4}`
+	ipv6re        = `(([a-f0-9]{0,4}[:]){1,8}([0-9a-fA-F]{1,4}|::)+)`
```
Comment (sairameshv) [inline pkg/obfuscator/ip.go:26]: Could you fix the linting issue?

**Review by sairameshv — CHANGES_REQUESTED** [review]

**Comment by sairameshv** [comment]
Please squash the change into a single commit.

**Comment by ansvu** [comment]
squash into one commit is done. Please check. Thanks.

**Review by sairameshv — APPROVED** [review]

**Comment by sairameshv** [comment]
LGTM

**Pull request merged**